pkg/tfbridge/info.go

```diff
@@ -293,7 +293,7 @@ type JavaScriptInfo struct {
 type PythonInfo struct {
 	Requires      map[string]string // Pip install_requires information.
 	Overlay       *OverlayInfo      // optional overlay information for augmented code-generation.
-	UsesIOClasses bool              // Indicates whether the package generates input/output classes.
+	UsesIOClasses bool              // Deprecated: No longer required, all providers use IO classes.
```

Comment:
Should I just remove this instead of deprecating? It'll mean forced action to remove this from `resources.go` on the next provider update.

Reply:
Actually, I think it makes sense to keep it deprecated rather than removing right away since that would break downstream tests.